WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
```

Comment:
- I like the name a lot, but if you consider it to be really misleading you can make a PR with the change
- Nothing to add to Jeremy's comment
- Sure, anyway this method is short and clear I think.
- Also agree with Jeremy. I just think that people wouldn't look into this file to find the documentation and it's pretty straightforward to use this method.